wire-swift-generator/src/main/java/com/squareup/wire/swift/SwiftGenerator.kt

```diff
@@ -186,7 +186,8 @@ class SwiftGenerator private constructor(
       if (type == ProtoType.ANY) return false
       if (isMessage) {
         val messageType = schema.getType(type!!) as MessageType
-        return messageType.supportsEmptyInitialization
+
+        return messageType.supportsEmptyInitialization && messageType.fields.isNotEmpty()
```

Comment:
This is the actual fix/change. I believe it is the correct fix overall if for any reason we cannot load the `fields` of a message we do not know whether we can provide a default value for it.